src/style.css

```diff
@@ -0,0 +1,92 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  height: 60px;
+  padding: 0 50px;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  align-self: center;
+  width: 40px;
+  height: 40px;
+
+}
+/* #region nav */
+.nav {
+  display: flex;
+
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
```

Comment:
if you want to remove default values like margin and padding remove them for ALL tags, use ul for this instead of a specific class